test/test_args.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+
+from cli import args
+
+
+class ArgsTest(unittest.TestCase):
+
+    def test_parse_key_value_1_parameter(self):
+        # when
+        result = args.parse_string_int_key_value_argument("key=1")
+        # then
+        self.assertEqual({"key": 1}, result)
+
+    def test_parse_key_value_2_parameters(self):
+        # when
+        result = args.parse_string_int_key_value_argument("key=1,number=2")
+        # then
+        self.assertEqual({"key": 1, "number": 2}, result)
```

Comment:
would it be easy to change to expect `1s`/`23m`/`1h 20m`/`10h` instead of an integer when users would need to calculate what's 6h would be in seconds? 😅

Reply:
`1h 20m` maybe an overkill, so maybe we support `[1-9][0-9]{,2}[smh]`? I.e. 1-999s/m/h

Reply:
we can create a separate ticket if it's a real problem to calculate 6 * 60 * 60. Didn't want to spend a lot of time for implementing parsing and covering implementation with tests for not frequently used cli argument.